README.md

```diff
@@ -29,17 +29,30 @@ Code::Blocks SVN rev 9843, because there are some changes in the Scripted wizard
 ## Install
 
 Certain files should be placed or symlinked inside the User data folder of Code::Blocks.
-On Linux this is /home/$USER/.local/share/codeblocks
+On Linux this is /home/$USER/.local/share/codeblocks and ~/.local/share/codeblocks
 
 1. You need to install AVR compiler (AVR GCC) and set it up inside Code::Blocks
-2. You should create a symlink for the wizard. Link the root of this repo to ~/.local/share/codeblocks/templates/wizard/arduino
+2. On Linux, you should create a symlink for the wizard. Link the root of this repo to ~/.local/share/codeblocks/templates/wizard/arduino
 
+On Windows; copy the files to %APPDATA%\CodeBlocks\share\codeblocks\templates\wizard\arduino folder
+   ```
+   mkdir %APPDATA%\CodeBlocks\share\codeblocks\templates\wizard\arduino
+   xcopy /Y wizard.script  "%APPDATA%\CodeBlocks\share\codeblocks\templates\wizard\arduino\"
+   xcopy /Y wizard.xrc     "%APPDATA%\CodeBlocks\share\codeblocks\templates\wizard\arduino\"
+   xcopy /Y *.png          "%APPDATA%\CodeBlocks\share\codeblocks\templates\wizard\arduino\"
+   xcopy /Y *.md           "%APPDATA%\CodeBlocks\share\codeblocks\templates\wizard\arduino\"
+   xcopy /Y /S files       "%APPDATA%\CodeBlocks\share\codeblocks\templates\wizard\arduino\files\"
+   ```
```

Comment:
I think I prefer if the instructions just tell the users to checkout the repo directly into the wizard/arduino folder on windows.